### DISCUSSION TITLE
This user account is blocked.

### DISCUSSION THREAD

**Discussion by DeveloperAronK** [vercel/community#4202] [Account Issue] [answered]
### Question

I can't able to signup with my personal account, I'm not doing any business on my account.
<img width="702" alt="Screenshot 2023-09-22 at 3 04 21 PM" src="https://github.com/vercel/community/assets/145562881/4bf6e534-2e48-4d09-a575-ea975ac22b86">

**Comment by ghost** [comment]
I'm getting the same error.

**Comment by maktubz1** [comment]
I can't able to signup with my personal account.

**Reply by hungchihsueh** [reply]
I can't able to signup with my personal account.

**Comment by hungchihsueh** [comment]
I can't able to signup with my personal account.

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
